go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/env0/terraform-provider-env0
 
-go 1.14
+go 1.16
```

Comment:
Probably you should also add a comment here...

Reply:
a221df7

Reply:
@omry-hay please re-review with 1a80ab6